.circleci/config.yml

```diff
@@ -597,7 +597,7 @@ jobs:
     docker:
       - image: alpine:3.11
     steps:
-      - run: apk add --no-cache --no-progress git bash curl ncurses jq
+      - run: apk add --no-cache --no-progress git bash curl ncurses jq openssh
```

Comment:
`openssh` will install the server as well, I think `openssh-client` will be slightly faster to install.

Reply:
A few seconds still counts :D